chapters/en/slides/module2/module2_13.Rmd

```diff
@@ -56,33 +49,129 @@ both within a bar and between bars.
 
 ---
 
-## Faceting creates subplots for different subsets of the data
+## Stacked histograms make it hard to compare between groups
 
 ```{python}
 alt.Chart(gm2018).mark_bar().encode(
     x=alt.X('life_expectancy', bin=alt.Bin(maxbins=30)),
     y='count()',
-    color='region'
-).properties(width=200, height=200
-).facet('region')
+    color='region')
+```
+
+Notes:
+We have previously seen how we can explore categorical dataframe columns
+by encoding them as the colour channel of our plots.
+
+Altair creates a stacked bar chart by default
+when we when encode a dataframe variable as the `color` channel.
+
+Just like with the stacked area chart,
+this is good when the total height of each bar
+is the most important,
+but it is not ideal when the main focus of our visualization
+is to compare the colored groups against each other.
+
+The reason it is difficult to compare
+the length of the coloured segments against each other
+(both within a bar and between bars),
+is that they don't share the same baseline
+so we can't just compare the position of the top part of the bars,
+but have to try to estimate their lengths.
+
+For these reasons,
+it is difficult to tell the difference between the regions
+in this plot
+and it is not an effective visualization.
+
+---
+
+## Layered histograms are also difficult to compare
+
+```{python}
+alt.Chart(gm2018).mark_bar(opacity=0.7).encode(
+    alt.X('life_expectancy', bin=alt.Bin(maxbins=30)),
+    alt.Y('count()', stack=False),
+    color='region')
+```
+
+Notes:
+If we tell Altair not to stack the bar along the y-axis,
+it will instead layer them behind each other.
+To be able to see all groups,
+we need to add some opacity to the bar mark.
+
+Although the bars share the same baseline here,
+they are still difficult to compare against each other,
+because there is so much overlap with different colors.
+
+Layered histograms and bar charts can be effective
+when there are few groups and clear separation between them,
+but that is not the case here
+and this plot is even harder to interpret than the previous one.
+
+---
+
+## Creating a separate subplot for each regions facilitate comparisons between them
+
+```{python}
+(alt.Chart(gm2018).mark_bar().encode(
+    x=alt.X('life_expectancy', bin=alt.Bin(maxbins=30)),
+    y='count()',
+    color='region')
+ .properties(width=200, height=150)
+ .facet('region'))#.configure_view(stroke=None)
```

Comment:
This one errors when converting to md

![image](https://user-images.githubusercontent.com/4560057/110191849-8eafd880-7ddf-11eb-89b4-b3c2fc89ac40.png)